### PR TITLE
DBR-321 - Fix issue with browser language (tests and refactoring)

### PR DESCRIPTION
**Pull request by andresmoschini**
Hi team,

As I promised in #221, I am adding tests and some minor refactoring related to language behavior.

Based on a @fgchaio comment, I am also changing the default language for Spanish.

Could you review it?

### CODE REVIEW

File: src/App.test.js [hunk 1]
```diff
@@ -90,6 +90,25 @@ describe('App component', () => {
...
+      // Arrange
+      const dependencies = {
+        sessionManager: createDoubleSessionManager(),
+      };
+
+      // Act
+      const { getByText } = render(
+        <AppServicesProvider forcedServices={dependencies}>
+          <Router initialEntries={['/login']}>
+            <App locale="fr" />
```
Comment (cbernat) [inline src/App.test.js]: 👍

File: src/i18n/utils.js [hunk 2]
```diff
@@ -12,3 +12,11 @@ export function flattenMessages(nestedMessages, prefix = '') {
...
   }, {});
 }
+
+export const defaultLanguage = 'en';
+
+export const availableLanguageOrNull = (lang) =>
+  (lang && ['es', 'en'].includes(lang) && lang) || null;
+
+export const availableLanguageOrDefault = (lang) =>
+  availableLanguageOrNull(lang) || defaultLanguage;
```
Comment (cbernat) [inline src/i18n/utils.js]: Why not have all in one function? `availableLanguageOrNull` as part of `availableLanguageOrDefault`

Comment (andresmoschini) [reply src/i18n/utils.js]: Because in a place I need _null_ result. I did not analyze if it is really necessary (I think so) or not.

Comment (cbernat) [reply src/i18n/utils.js]: You think you might need it, ok.

**Comment by cbernat** [comment]
👍

File: src/i18n/utils.js (hunk 2 above)
Comment (andresmoschini) [reply src/i18n/utils.js]: Here: https://github.com/FromDoppler/doppler-webapp/pull/222/files#diff-14b1e33d5bf5649597cdc0e4f684daddR48

**Pull request merged**